vehicles/src/modules/special-auth/loa.controller.ts

```diff
@@ -157,14 +156,14 @@ export class LoaController {
       }),
     )
     file: Express.Multer.File,
-    @Body() updateLoaFileDto: UpdateLoaFileDto,
+    @Body() createLoaFileDto: CreateLoaFileDto,
   ): Promise<ReadLoaDto> {
     const currentUser = request.user as IUserJWT;
     const loa = await this.loaService.updateLoa(
       currentUser,
       companyId,
       loaId,
-      updateLoaFileDto?.body,
+      createLoaFileDto?.body,
```

Comment:
Use updateLoaFileDto. It can simply extend createLoaFileDto

Reply:
done